src/app/settings/initial-settings.ts

```diff
@@ -42,4 +42,13 @@ export const initialSettingsState: Settings = {
     'customstat',
     'notes',
   ],
+  loStatSortOrder: [
```

Comment:
These initial settings seem to be in both dim-api and here. Do we need them both? I am thinking I may be able to delete these if not.

Reply:
What I've been doing is to declare new settings here so I can iterate, then move them to the dim-api types library once it launches.